### PR TITLE
Remove all traces of Person class and refactor AddressBook to TeachingAssistant

### LINKED ISSUE #217: Make Teaching Assistant use contact instead of person

### LINKED ISSUE #218: Delete all traces of person class

### LINKED ISSUE #224: Refactor names to Teaching Assistant

### PR DESCRIPTION
**Pull request by tsh22**

### CODE REVIEW

**Review by lrj689 — APPROVED** [review]

File: src/main/java/seedu/address/logic/commands/AddContactCommand.java [hunk 1]
```diff
@@ -1,67 +1,64 @@
...
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
+            + PREFIX_NAME + "Bob Choo "
+            + PREFIX_PHONE + "22222222 "
+            + PREFIX_EMAIL + "bob@example.com "
             + PREFIX_TAG + "friends "
             + PREFIX_TAG + "owesMoney";
 
-    public static final String MESSAGE_SUCCESS = "New person added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+    public static final String MESSAGE_SUCCESS = "New contact added: %1$s";
```
Comment (nicholastanvis) [inline src/main/java/seedu/address/logic/commands/AddContactCommand.java:34]: is it also possible to move these into messages and import statically?

Comment (nicholastanvis) [inline src/main/java/seedu/address/logic/commands/AddContactCommand.java:34]: is it also possible to move these into messages and import statically?

File: src/test/data/JsonSerializableAddressBookTest/typicalTeachingAssistant.json [hunk 2]
```diff
@@ -4,62 +4,62 @@
...
+    "tagged" : ["colleagues"]
   }, {
     "name" : "David Meier",
     "phone" : "87652533",
     "email" : "cornelia@example.com",
-    "tagged" : [ "friends" ]
+    "tagged" : ["friends"]
   }, {
-    "name" : "Elle Meyer",
+    "name" : "Ellie Meyer",
```
Comment (nicholastanvis) [inline src/test/data/JsonSerializableAddressBookTest/typicalTeachingAssistant.json:24]: wat lel

File: src/test/java/seedu/address/logic/commands/ClearCommandTest.java [hunk 3]
```diff
@@ -1,7 +1,7 @@
 package seedu.address.logic.commands;
 
 import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
-import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalContacts.getTypicalContactsAddressBook;
```
Comment (nicholastanvis) [inline src/test/java/seedu/address/logic/commands/ClearCommandTest.java]: is it possible to make use of the existing gettypicalteachingassistant

**Review by goatygoatygoat — APPROVED** [review]

File: src/test/data/JsonSerializableAddressBookTest/typicalTeachingAssistant.json (hunk 2 above)
Comment (tsh22) [reply src/test/data/JsonSerializableAddressBookTest/typicalTeachingAssistant.json:24]: For testing purposes haha

**Review by goatygoatygoat — APPROVED** [review]

**Review by nicholastanvis — APPROVED** [review]

**Pull request merged**